vllm/sequence.py

```diff
@@ -595,9 +596,11 @@ def __init__(
         encoder_seq: Optional[Sequence] = None,
         trace_headers: Optional[Mapping[str, str]] = None,
         prompt_adapter_request: Optional[PromptAdapterRequest] = None,
+        priority: Optional[int] = None,
```

Comment:
Perhaps it would be better to define a default priority of `0`? Then it's clearer how non-prioritized requests will be handled in relation to prioritized ones.

Reply:
Keeping default as None ensures consistency with other parameters

Reply:
I'm not sure about this, I don't think the fact that the default for some of the other params is so relevant.

I feel it would be better to be explicit about that the default priority is 0, so it's clear how unprioritized requests interact with prioritized ones. Otherwise we are dependent on some arbitrary ordering between `None` and `int`.

Reply:
I think it would make sense to change the type to just be `int` rather than `Optional[int]` everywhere, and wherever it's an arg have the default be 0.